app/Http/helpers.php

```diff
@@ -49,7 +49,9 @@ function dosomething_global_countries()
  */
 function should_collect_phone()
 {
-    return in_array(get_country_code(), dosomething_global_countries());
```

Comment:
Was this the only place we used `dosomething_global_countries()`? If so, we should remove it.